WordPress/wordcamp.org
# Use CLDR for country code and name data

This adds the [WP CLDR plugin](https://wordpress.org/plugins/wp-cldr/) to the WordCamp codebase for use in places where we need a list of country codes and/or names, replacing the hard-coded ISO-3166 list we had previously.

See https://meta.trac.wordpress.org/ticket/4638 for details and reasoning behind this change.

Note that we won't actually activate the WP CLDR plugin network-wide or on individual sites, because we don't need or want the [WP Admin UI](https://ps.w.org/wp-cldr/assets/screenshot-1.png?rev=1375816) that ships with the plugin. Instead, the `wcorg_get_countries` function loads the necessary class file directly.

**To test**

* This PR doesn't actually add the wp-cldr plugin directly. You can either run `composer update` to install it, or just do it manually from your testing environment of choice. Don't activate the plugin after installing it, though.
* Visit a page that outputs a `<select>` element containing all the countries. `https://central.wordcamp.org/wordcamp-organizer-application/` is a good choice if you're using your sandbox.
* The `<select>` should contain all the countries, with some naming differences compared to what is currently on Production (see the Meta ticket for specific examples).

**Pre-deploy tasks**

* [x] Add the wp-cldr plugin to our list of plugin externals
* [x] Add the `cldr-countries` feature flag
* [x] Make sure WC Central doesn't haven't the flag

**Post-deploy tasks**

* [x] Update the country field for all Multi Event Sponsors' contact info
